workloads/templates/common-templates.md

```diff
@@ -131,3 +131,46 @@ oc process rhel7-generic-tiny PVCNAME=mydisk NAME=rheltinyvm | oc apply -f -
 
 Please note that, after the generation step, VM objects and template objects have no relationship with each other besides the aforementioned label (e.g. changes
 in templates do not automatically affect VMs, or vice versa).
+
+## common template customization
+
+### template customization - memory, CPU
+
+There are three options to customize VM memory and CPU:
+
+ * select flavor - tiny, small, medium, etc. Each flavor grants to VM different amount of RAM and CPU cores.
+ * setting up directly editable fields spec.template.spec.domain.cpu.cores and spec.template.spec.domain.resources.requests.memory, example:
```

Comment:
Please make some of the references into links - the editable fields for example should point to the part that describes what it is and what is the format. Because you can customize the list of editable fields as well.